src/main/java/org/zone/region/flag/entity/player/move/message/greetings/GreetingsFlagType.java

```diff
@@ -30,12 +34,25 @@
 
     @Override
     public @NotNull GreetingsFlag load(@NotNull ConfigurationNode node) throws IOException {
-        String message = node.node("Message").getString();
+        String message = node.node("GreetingsMessage").getString();
+        String displayType = node.node("GreetingsMessageDisplayType").getString();
         if (message == null) {
-            throw new IOException("Couldn't get message");
+            throw new IOException("Could not get message");
+        }
+        if (displayType == null) {
+            throw new IOException("Couldn't get the display type");
```

Comment:
To maintain backwards compatibility with 1.0.0, we cannot fail the load because they don't have the display type. Try just suppling the default display type (such as the issue states)

Reply:
k

Reply:
Still not right

Reply:
whyyyyyyy

Reply:
As mentioned to you on discord, you are setting the value in the config, but not the variable, meaning that the DisplayType object would never be found. 

On a side note, you dont need to set it in the config if its not there, loading the flag shouldnt really be saving anything, leave that up to the saving of the value